### ISSUE TITLE
Avoid Audacious overwriting/replacing current playlist with imported one

### ISSUE THREAD

**Issue by slrslr** [audacious-media-player/audacious#1508]
**Description**
Inside possibly older Audacious 4.2 on Debian 12 (stable) Linux with KDE, when I switch to a certain playlist tab, then select Playlist menu, Import ..., select .pls file, it cause replacement of all the playlist entries by the entries from newly imported playlist. This is very bad idea, if one do not often save the playlist, he can loose items. One would expect that there will be new playlist added or that Audacious will ask what to do before replacing.

Maybe this issue has already been fixed. Can anyone reproduce this please?

**Comment by radioactiveman** [comment]
Yes, I can reproduce this behavior. It is the same with the latest version (4.4.2).

@jlindgren90: Is this intentional? Even when you rename the playlist from "Now playing" its entries are overwritten immediately.

**Comment by jlindgren90** [comment]
I think this is the way it has been as long as I can remember, but I see the concern. I don't have a problem with changing the behavior to append to the current playlist instead, if you think that makes sense.

**Comment by jlindgren90** [comment]
Alternatively we could get rid of "Import Playlist" since "Add Files" can already do the same thing.

**Comment by slrslr** [comment]
I do not think it should mess with the current playlist. You are opening NEW(separate) playlist. So it should open it on a new tab IMO. User can then move content between the playlists.

**Comment by jlindgren90** [comment]
That could also work
